ament_pep257/ament_pep257/main.py

```diff
@@ -107,6 +107,13 @@ def main(argv=sys.argv[1:]):
     return rc
 
 
+def _filename_in_excludes(filename, excludes):
```

Comment:
Just a suggestion: something a bit more Pythonic could be

```python
def _filename_in_excludes(filename, excludes):
    return any([os.path.abspath(filename).startswith(e) for e in excludes])
```

:smiley:

Reply:
Agree, thank you

Reply:
In fact, since it is one-liner, there is no need in function at all. It may be inlined in the places of use

Reply:
Amended.

> In fact, since it is one-liner, there is no need in function at all. It may be inlined in the places of use
Though I decided to keep function instead of copy-paste.

Reply:
My bad, this is actually worse than your original function! [Using a generator expression](https://stackoverflow.com/a/16505590/6476709) (without the `[]`) is better than using a list comprehension.